grammar.js

```diff
@@ -331,6 +332,16 @@ module.exports = grammar({
       ')',
     ),
 
+    each_loop: $ => seq(
+      field('args', $._expression),
```

Comment:
Perhaps this needs to be more granular... @murtaza64  wdyt?

Reply:
This syntax just seems like a function call. I've tried to implement function calls with optional parentheses but they've given me a huge headache in the past. Does something like
```
collection.each { x -> doSomething(x) }
```
not get parsed properly?

Reply:
happy to have it be a function call. it errored on the dot operator after a list... having some PTO atm but happy with what you come up with or i can take another stab at in early September.

Reply:
Ok, I've just added `list` to `_prefix_expression` and reworked the test case... Hopefully that should be enough.